gtsam/hybrid/HybridGaussianFactorGraph.cpp

```diff
@@ -323,18 +323,29 @@ hybridElimination(const HybridGaussianFactorGraph &factors,
     // If there are no more continuous parents, then we create a
     // DiscreteFactor here, with the error for each discrete choice.
 
-    // Integrate the probability mass in the last continuous conditional using
-    // the unnormalized probability q(μ;m) = exp(-error(μ;m)) at the mean.
-    //   discrete_probability = exp(-error(μ;m)) * sqrt(det(2π Σ_m))
-    auto probability = [&](const Result &pair) -> double {
+    // Compute the probability q(μ;m) = exp(-error(μ;m)) * sqrt(det(2π Σ_m))
```

Comment:
Oh, I guess this will conflict with my simplified PR...

Reply:
Yeah we should probably land all of my old PRs so we can reduce conflicts.